build.gradle

```diff
@@ -180,8 +180,11 @@ allprojects {
             }
         }
     }
+
```

Comment:
It doesn't look like a necessary change here.

Reply:
tnx